README.md

```diff
@@ -615,3 +571,184 @@ if resp.IsSuccessful {
     fmt.Printf("Permission Current: %v", resp.PermissionCurrentResponse)
 }
 ```
+
+## Verify
+[Verify API Reference](hhttps://docs.fullcontact.com/reference/activity)
+- `verify.activity`
+- `verify.match`
+- `verify.signals`
+
+### Verify Request
+Verify accepts a `MultifieldRequest` as its input for
+
+- `verify.activity`
+- `verify.match`
+- `verify.signals`
+
+All Verify APIs requires `MultifieldRequest` requests, which can be constructed by using `NewMultifieldRequest` and following are it's parameters.
```

Comment:
Lets create a section for MultiFieldRequest, and refer it here (and in Person and Resolve requests), so there is less duplicate. You can refer Java client lib Readme.

Reply:
@prakhar-j  Is this the java lib? https://github.com/fullcontact/fullcontact-java-client

Didn't see any section of MFR in this repo

Reply:
Should be https://github.com/fullcontact/fullcontact-java-client/tree/master/java11#multifieldrequest and https://github.com/fullcontact/fullcontact-java-client/tree/master/java11#building-a-person-enrichresolve-request

Reply:
go client readme updated to mirror the docs for java client